examples/models/tf.py

```diff
@@ -0,0 +1,105 @@
+# sources copied/modified from https://github.com/tensorflow/models/blob/master/samples/core/get_started/
```

Comment:
We should provide requirements.txt so that people can pip install and run this script to get a tf model

Reply:
what do you mean? this file has no deps

Reply:
tensorflow and sklearn

Reply:
ah right...checked out